### ISSUE TITLE
talkback no longer reads the contents of a conversation in gmail

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#363]
```
What steps will reproduce the problem?
1. Open a conversation from any mail box in gmail.

2. Attempt to read the screen with talkback.

3. Talkback reads the bar at the top of the screen, but doesn't read the text 
of the conversation.


What is the expected output? What do you see instead?
I expect talkback to read all the contents displayed on the screen.  If I start 
reading the screen using read from top or read from next, I expect talkback to 
start reading the contents of the screen and continue reading until the end of 
the conversation.
Talkback only reads the menu bar at the top of the screen and doesn't read any 
other part of the gmail screen.

What version of the product are you using? On what operating system?
gmail version 4.7.2-96715
talkback version 3.5.1
Android version 4.1.2


Please provide any additional information below.


```

Original issue reported on code.google.com by `hittsj...@gmail.com` on 4 May 2014 at 12:54

**Comment by GoogleCodeExporter** [comment]
```
This issue is now fixed.  I would like to have this issue closed and removed 
from the list of issues.
```

Original comment by `hittsj...@gmail.com` on 1 Oct 2014 at 10:43
